src/odemis/acq/stitching/_registrar.py

```diff
@@ -460,6 +460,8 @@ def __init__(self):
         self.shifts_hor = [[None]]
         self.shifts_ver = [[None]]
 
+        self.registered_positions = [[(None, None)]]  #list of calculated positions (tuple of 2 floats/None)
+
```

Comment:
Is there a need to create the list here for the field corrections code? Otherwise I think it is enough to create it in line 504 where you overwrite it.

Reply:
I thought it is nice to define this list at the init because the convention is more or less to declare it there and explain what the variable holds.

Reply:
Then add the shape (x, y).

Reply:
I also think it's a good idea to initialise all the attributes in the init. However, here I'd suggest to just initialise it to `None`, as it just would be a bad idea to read it before calling `getPositions()`.
Also, please fix the type. It's *not* a list. See the docstring of `_assemble_mosaic()`.

Reply:
Changed it to numpy.array (Docstring of _assemble_mosaic is actually wrong currently)